ui-admin/src/study/adminTasks/DeleteStudyModal.tsx

```diff
@@ -0,0 +1,62 @@
+import React, { useState } from 'react'
+import Modal from 'react-bootstrap/Modal'
+import LoadingSpinner from 'util/LoadingSpinner'
+import Api from 'api/api'
+import { Store } from 'react-notifications-component'
+import { failureNotification } from '../../util/notifications'
+import { Study } from '@juniper/ui-core/build/types/study'
+import { Portal } from '@juniper/ui-core/build/types/portal'
+
+/** renders a modal that allows deleting a survey */
+const DeleteStudyModal = ({
+  portal, study, onDismiss, reload
+}: {
+  portal: Portal, study:Study, onDismiss: () => void, reload: () => void}) => {
+  const [isLoading, setIsLoading] = useState(false)
+  const [confirmDeleteStudy, setConfirmDeleteStudy] = useState('')
+  const deleteString = `delete ${study.name}`
+  const canDelete = confirmDeleteStudy.toLowerCase() === deleteString.toLowerCase()
+
+  const deleteStudy = async () => {
+    setIsLoading(true)
+
+    await Api.deleteStudy(portal.shortcode, study.shortcode).catch(() =>
+      Store.addNotification(failureNotification('Error deleting study'))
+    )
+    reload()
+    setIsLoading(false)
+    onDismiss()
```

Comment:
we have a helper function that handles error/loading stuff.  So you can simplify the above to:
```
await doApiLoad( async () => Api.deleteStudy(portal.shortcode),
{ setIsLoading} )
reload()
onDismiss()
```